src/mds/CDir.cc

```diff
@@ -194,9 +194,9 @@ CDir::CDir(CInode *in, frag_t fg, MDCache *mdcache, bool auth) :
   cache(mdcache), inode(in), frag(fg),
   first(2),
   dirty_rstat_inodes(member_offset(CInode, dirty_rstat_item)),
-  projected_version(0),
   dirty_dentries(member_offset(CDentry, item_dir_dirty)),
   item_dirty(this), item_new(this),
+  projected_version(0),
```

Comment:
@varshar16 As a follow-up change (commit), please also move these trivial initializations into the header for the `bool`/`int` and other scalar types. See also [1].

[1] https://en.cppreference.com/w/cpp/language/data_members

Reply:
Done